graderutils/tracebackformat.py

```diff
@@ -127,10 +137,17 @@ def clean_feedback(result_groups, config):
         # Now for the full output from running the test suite
         for task in config:
             if task.get("hide_tracebacks", False) and not task.get("hide_tracebacks_short_only", False):
+                remove_sentinel = task.get("remove_sentinel", '')
+
+                # remove_more_sentinel has been deprecated
+                if task.get("remove_more_sentinel", ''):
```

Comment:
If I understand correctly, this change makes `remove_more_sentinel` behave differently than before. I.e. it doesn't require `hide_tracebacks` to be used. The idea is that we _shouldn't_ break existing behaviour, unless current behaviour is clearly a bug. In this case, course could include `remove_more_sentinel` and expect it to be ignored when `hide_tracebacks` is false.

It's bit hard for me to follow the code exactly, but I guess it would be enough to change this to 

```python
remove_more_sentinel = task.get('remove_more_sentinel')
if hide_tracebacks and remove_more_sentinel and not remove_sentinel:
    remove_sentinel = remove_more_sentinel
```

Reply:
The above line (149) requires hide_tracebacks to be set to `true`, so behaviour isn't changed:
https://github.com/Aalto-LeTech/python-grader-utils/blob/7baddd5e6255ccbc0868d9a871b29a6e7155cfbc/graderutils/tracebackformat.py#L139

Reply:
I first read that one of these wasn't inside that test. my bad.